lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -175,8 +179,8 @@ static Node *simplifyNode(Node *node, Function *F) {
   SIMPLIFY_OPERANDS(Sub)
   SIMPLIFY_OPERANDS(Max)
   SIMPLIFY_OPERANDS(Min)
-  SIMPLIFY_OPERANDS(CmpLTE)
-  SIMPLIFY_OPERANDS(CmpEQ)
+  // SIMPLIFY_OPERANDS(CmpLTE)
```

Comment:
this will be brought back in the PR for review.

Reply:
It's actually already removed in master since I landed the `simplifyNode()` redesign, so if you rebase it will go away anyway after you fix the conflicts 😃